stubs/setuptools/setuptools/_scripts.pyi

```diff
@@ -0,0 +1,46 @@
+from _typeshed import Incomplete, Unused
```

Comment:
This is private. Unless we need to reference it somewhere, better off omitting it.

Reply:
fixed, but maybe this will be used in future. (i'm not big fan of hiding private things:)

Reply:
Generally we don't include non-public interfaces, unless requested.

---

I can tell you specific to this file:
`setuptools/command/easy_install.py` is only kept temporarily for pbr compatibility
Everything else should be exposed through `setuptools/command/install_scripts.py`

Any newer project shouldn't be using `_scripts`.
Any older project that have a hack / unsupported use-case where they need to update an import to `_scripts` are likely to also be having typing issues (namely with `distutils`)

Reply:
thank you for the detailed explanation :)

Reply:
Thank you for the updates!